src/pages/components/OperatorSelector.tsx

```diff
@@ -6,14 +6,26 @@ import { Autocomplete, TextField } from '@mui/material'
 type OperatorSelectorProps = {
   operatorId?: string
   setOperatorId: (operatorId: string) => void
+  onlyMajorOperators?: boolean
 }
 
-const OperatorSelector = ({ operatorId, setOperatorId }: OperatorSelectorProps) => {
+const OperatorSelector = ({
+  operatorId,
+  setOperatorId,
+  onlyMajorOperators = false,
+}: OperatorSelectorProps) => {
   const [operators, setOperators] = React.useState<Operator[]>([])
-
+  console.log(operatorId)
```

Comment:
It is not mandatory'but, this console statement can be removed

Reply:
Yup, I forgot to remove it. Thanks :)